pkg/asset/installconfig/installconfig.go

```diff
@@ -122,7 +122,7 @@ func (a *InstallConfig) Load(f asset.FileFetcher) (found bool, err error) {
 func (a *InstallConfig) finishAWS() error {
 	// Set the Default Edge Compute pool when the subnets in AWS Local Zones are defined,
 	// when installing a cluster in existing VPC.
-	if len(a.Config.Platform.AWS.Subnets) > 0 {
+	if len(a.Config.Platform.AWS.VPC.Subnets) > 0 {
```

Comment:
Since `VPC` is an embeded struct, we can just use `a.Config.Platform.AWS.Subnets` (i.e. no changes to existing code). 

That said, I am using `.VPC.Subnets` explicitly here to emphasise the new field. Please let me know if you prefer the above shorter way.

Reply:
Will use shorter version! This makes the PR size small and less unnecessary changes.